syntax = "proto3";

package NStrm.NPackager.NResolveDataProto;

message TData {
    int64 Version = 1; // counter
    repeated TEndpointSetInAllClusters Id2Endpoints = 2;
}

message TEndpointSetInAllClusters {
    string EndpointSetId = 1;
    int64 Version = 2; // counter
    repeated TEndpointSet Cluster2EndpointSet = 3;
}

message TEndpointSet {
    string Cluster = 1;
    int64 Version = 2; // timestamp of TRspResolveEndpoints
    repeated bytes Addresses = 3;
}
